### PR TITLE
Nanopore fast5

### PR DESCRIPTION
**Pull request by tom114**
## Description of changes
Adding Nanopore's `.fast5` format to IRIDA's data storage system.

## Related issue
Related to #117. 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: doc/_includes/tutorials/common/samples/galaxy-export.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Navigate to the project that contains the samples that you’re interested in ex
 On this page you will fill in a few options about your Galaxy export:
 * Galaxy User Email: This email must exactly match the email of your Galaxy account.  Incorrectly adding your Galaxy email will result in a failed export.
 * Include assemblies: Whether to include assemlies in the Galaxy export.  If this box is checked, any assemblies associated with the samples in the cart will be included.
+* Include FAST5: Whether to include FAST5 data in the Galaxy export.  If this box is checked, any FAST5 associated with the samples in the cart will be included.
```
Comment (joshsadam) [inline doc/_includes/tutorials/common/samples/galaxy-export.md:26]: I think this should read `any FAST5 files associated` or something like that

Comment (tom114) [reply doc/_includes/tutorials/common/samples/galaxy-export.md]: Fixed in 3f73d98efd

File: src/main/webapp/resources/js/components/sample/SampleFileUploader.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react";
 import { setBaseUrl } from "../../utilities/url-utilities";
 import { FileUploader } from "../files/FileUploader";
+import { showNotification } from "../../modules/notifications";
```
Comment (joshsadam) [inline src/main/webapp/resources/js/components/sample/SampleFileUploader.jsx:4]: Unused import, can you remove it (my fault).

File: src/main/java/ca/corefacility/bioinformatics/irida/web/controller/api/projects/RESTProjectSamplesController.java [hunk 3]
```diff
@@ -287,11 +291,14 @@ private void addLinksForSample(final Optional<Project> p, final Sample s) {
 		s.add(linkTo(methodOn(RESTSampleSequenceFilesController.class).listSequencingObjectsOfTypeForSample(s.getId(),
```
[the remark this answers is not in the document]
Comment (tom114) [reply src/main/java/ca/corefacility/bioinformatics/irida/web/controller/api/projects/RESTProjectSamplesController.java:290]: Fixed the javadoc in fde753e

File: src/main/webapp/resources/js/components/sample/SampleFileUploader.jsx (hunk 2 above)
Comment (tom114) [reply src/main/webapp/resources/js/components/sample/SampleFileUploader.jsx]: Removed import in 7a53390

**Pull request merged**